GoogleForCreators/web-stories-wp
# Analytics: Update message when Site Kit is installed

## Context



## Summary


As of https://github.com/google/web-stories-wp/issues/6124, the message around Adsense is changed when when adsense is the sitekit plugin is enabled. But @swissspidy brings up a valid point in 
https://github.com/google/web-stories-wp/pull/7622#issuecomment-845401577 that the messaging / styling should also change with google analytics as well. 

For reference, here is new styling for adsense messaging. 
<img src="https://user-images.githubusercontent.com/237508/119018371-e9df7900-b993-11eb-9865-86a076ef30d3.png" />

## Relevant Technical Choices



## To-do



## User-facing changes

<img width="1884" alt="Screenshot 2021-05-24 at 16 01 43" src="https://user-images.githubusercontent.com/237508/119367683-1c49e880-bcaa-11eb-8171-70f89d853650.png">

## Testing Instructions



### QA






This PR can be tested by following these steps:

1. Activate sitekit plugin
2. Activate google analytics in sitekit. 
3.  Web stories > settings
4. See message matches screenshot

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #7672
